src/staff-administrator/components/staff-add-edit-user/staff-add-edit-user-form/staff-add-edit-user-form.component.spec.ts

```diff
@@ -5,25 +5,55 @@ import { Component, NO_ERRORS_SCHEMA } from '@angular/core';
 import { ReactiveFormsModule } from '@angular/forms';
 import { ActivatedRoute, Router } from '@angular/router';
 import { RouterTestingModule } from '@angular/router/testing';
-import { ExuiCommonLibModule, FilterService } from '@hmcts/rpx-xui-common-lib';
+import { ExuiCommonLibModule, FilterService, GroupOptions } from '@hmcts/rpx-xui-common-lib';
 import { of } from 'rxjs';
 import { StaffUserCheckAnswersComponent } from '../staff-user-check-answers/staff-user-check-answers.component';
 import { StaffAddEditUserFormComponent } from './staff-add-edit-user-form.component';
+import { StaffFilterOption } from '../../../models//staff-filter-option.model';
```

Comment:
Import ordering incorrect here, StaffFilterOption should be above StaffUserCheckAnswersComponent

Reply:
Updated.

Reply:
Sorry Aamir, what I meant was StaffFliterOption should be moved upwards, before the others. StaffAddEditUserFormComponent should be the last one in the list of imports because its relative path is shortest (hence why StaffFliterOption should be above both of these)